src/CMakeLists.txt

```diff
@@ -352,45 +371,20 @@ set(src_nupiccore_srcs
     nupic/utils/Watcher.cpp)
 
 set(src_lib_static_nupiccore_srcs
-    ${src_capnp_srcs}
+    ${src_capnp_generated_srcs}
     ${src_py_support_files}
     ${src_nupiccore_srcs})
 
-set(src_lib_static_nupiccore_solo_link_libs
```

Comment:
Don't need any of this stuff any longer. The relationship with the externals is much simpler now that they are wrapped in `add_library` targets.